### PR TITLE
add description about .clang-tidy

### PR DESCRIPTION
**Pull request by sumyapp**

### CODE REVIEW

**Comment by sumyapp** [comment]
.clang-tidy is YAML or JSON file.

> > Specifies a configuration in YAML/JSON format:
>                                      -config="{Checks: '*',
>                                                CheckOptions: [{key: x,
>                                                                value: y}]}"
>                                    When the value is empty, clang-tidy will
>                                    attempt to find a file named .clang-tidy for
>                                    each source file in its parent directories.
> https://clang.llvm.org/extra/clang-tidy/

**Comment by sumyapp** [comment]
I add a new commit, if it looks okay, please merge this PR. Thanks for your reviews.

File: docs/tools/cplusplus/clang-tidy.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ hide_title: true
 
 ## Getting Started
 
-To start using Clang-Tidy, enable it in your [repository settings](../../getting-started/repository-settings.md).
+To start using Clang-Tidy, enable it in your [repository settings](../../getting-started/repository-settings.md).  
+If you want to configure Clang-Tidy, please put the `.clang-tidy` file(YAML/JSON) in the root directory. You can disable/enable Clang-Tidy rules (Checks) via the file.  
+Ref: https://clang.llvm.org/extra/clang-tidy/
```
Comment (ybiquitous) [inline docs/tools/cplusplus/clang-tidy.md:22]:
```suggestion
To start using Clang-Tidy, enable it in your [repository settings](../../getting-started/repository-settings.md).

If you want to configure Clang-Tidy, please put the `.clang-tidy` file(YAML/JSON) in the root directory. You can disable/enable Clang-Tidy rules (Checks) via the file. See the [document](https://clang.llvm.org/extra/clang-tidy/) more details.
```

**Comment by ybiquitous** [comment]
After the change 7e0ed26:

image

**Comment by ybiquitous** [comment]
I'll merge this later if no oppositions.

**Pull request merged**